Pod/Classes/SEGAmplitudeIntegration.m

```diff
@@ -170,6 +170,13 @@ - (void)track:(SEGTrackPayload *)payload
 
 - (void)screen:(SEGScreenPayload *)payload
 {
+    if ([(NSNumber *)[self.settings objectForKey:@"trackAllPagesV2"] boolValue]) {
+        NSString *event = [[NSString alloc] initWithFormat:@"Loaded a Screen"];
```

Comment:
hmmm... why use format here? This should just be `NSString *event = @"Loaded a Screen"`.

Reply:
Good point